src/app/pages/sharing/iscsi/extent/extent-form/extent-form.component.ts

```diff
@@ -103,7 +109,7 @@ export class ExtentFormComponent {
     },
     {
       type: 'select',
-      name: 'iscsi_target_extent_blocksize',
+      name: 'blocksize',
       placeholder: T('Logical block size'),
       tooltip: T('Only override the default if the initiator requires a\
```

Comment:
`` Do not override the default unless the initiator requires a specific block size. ``